class/defaults.yml

```diff
@@ -8,13 +8,13 @@ parameters:
     charts:
       topolvm:
         source: https://topolvm.github.io/topolvm
-        version: v6.0.0
+        version: v8.0.0
 
     images:
       topolvm:
         registry: ghcr.io
         repository: topolvm/topolvm-with-sidecar
-        tag: 0.11.1
+        tag: 0.14.0
```

Comment:
Do we need explicitly define the tag? Looks to be `.Chart.AppVersion` already default in `Chart.yaml`.

Reply:
According to https://syn.tools/syn/explanations/commodore-components/container-images.html it best practice.  
But yes, I asked myself the same.